create-targz.sh

```diff
@@ -58,7 +58,7 @@ function build() {
   cp "${ORIGINDIR}"/linux_files/local.conf "${TMPDIR}"/dist/etc/fonts/
   cp "${ORIGINDIR}"/linux_files/00-remix.sh "${TMPDIR}"/dist/etc/profile.d/
   cp "${ORIGINDIR}"/linux_files/00-remix.fish "${TMPDIR}"/dist/etc/fish/conf.d/
-  chmod -x "${TMPDIR}"/dist/etc/profile.d/00-remix.sh
+  chmod -x,+r "${TMPDIR}"/dist/etc/profile.d/00-remix.sh
   chmod -x "${TMPDIR}"/dist/etc/fish/conf.d/00-remix.fish
```

Comment:
Probably it should be needed also for fish
```suggestion
  chmod -x,+r "${TMPDIR}"/dist/etc/fish/conf.d/00-remix.fish
```